### PR TITLE
add issues and pull request templates

### PR DESCRIPTION
**Pull request by seanknox**
This defines templates that will automatically populate the form when users and contributors open new issues or pull requests. The goal is to guide people as they file issues or contribute to the project.

Thanks @arschles for showing me how to create these!

---
This change is [Reviewable](https://reviewable.io/reviews/azure/acs-engine/655)

### CODE REVIEW

**Comment by seanknox** [comment]
@squillace thought you might want to view this too. It's some in-repo documentation to guide folks through the issue filing or PR process.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+If this is a FEATURE REQUEST, please:
+  - Describe *in detail* the feature/behavior/change you'd like to see.
+
+In both cases, be ready for followup questions, and please respond in a timely
+manner.  If we can't reproduce a bug or think a feature already exists, we
+might close your issue.  If we're wrong, PLEASE feel free to reopen it and
+explain why.
+-->
+
+**Orchestrator and version (e.g. Kubernetes, DC/OS, Swarm**
```
Comment (anhowe) [inline .github/ISSUE_TEMPLATE.md]: missing ')'

**Comment by dmitsh** [comment]
@acs-bot test this please

**Pull request merged**